### PR TITLE
Improve wording

### PR DESCRIPTION
**Pull request by vaijab**
## Description

Changes proposed in this pull request:

- Improve working of app for Slack

### CODE REVIEW

**Review by madebyrogal — COMMENTED** [review]
LGTM.
But what about older and current docs version. Don't you fix them too?

**Comment by vaijab** [comment]
> But what about older and current docs version. Don't you fix them too?

I thought about that. Do we need to for the app for Slack submission review? @pkosiec

**Comment by pkosiec** [comment]
@vaijab I think the changes for v1.11 + latest will be enough 👍 So feel free to merge it now as I'll release botkube docs very soon 👍

File: docs/configuration/helm-chart-parameters.md
Comment (pkosiec) [inline docs/configuration/helm-chart-parameters.md:1]: This is fetchedfrom the https://github.com/kubeshop/botkube/blob/main/helm/botkube/README.md file, which is generated from https://github.com/kubeshop/botkube/blob/main/helm/botkube/values.yaml comments. Fortunately I have my PR open so I can (and will) synchronize your changes there 👍

**Review by pkosiec — APPROVED** [review]

File: docs/configuration/helm-chart-parameters.md
Comment (pkosiec) [reply docs/configuration/helm-chart-parameters.md:1]: Here it is: https://github.com/kubeshop/botkube/pull/1445

**Pull request merged**